code/modules/uplink/uplink_items.dm

```diff
@@ -990,39 +1102,129 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
+			in addition to dealing serious damage to nearby personnel."
+	item = /obj/item/grenade/syndieminibomb
+	cost = 6
+	exclude_modes = list(/datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/explosives/gluon
+	name = "Gluon Grenade"
+	desc = "A highly advanced grenade that releases a harmful stream of radioactive gluons around the point of detonation. \
+			The gluons seriously cool the surrounding area and tire anyone hit from the impact."
+	item = /obj/item/grenade/gluon
+	cost = 10
+	include_modes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/explosives/bombanana
+	name = "Bombanana"
+	desc = "A banana with an explosive taste! discard the peel quickly, as it will explode with the force of a syndicate minibomb \
+		a few seconds after the banana is eaten."
+	item = /obj/item/reagent_containers/food/snacks/grown/banana/bombanana
+	cost = 4 //it is a bit cheaper than a minibomb because you have to take off your helmet to eat it, which is how you arm it
+	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/explosives/bioterrorfoam
+	name = "Bioterror Foam Grenade"
+	desc = "A powerful chemical foam grenade which creates a deadly torrent of foam that will mute, blind, confuse, \
+			mutate, and irritate carbon lifeforms. Specially brewed by Tiger Cooperative chemical weapons specialists \
+			using additional spore toxin. Ensure suit is sealed before use."
+	item = /obj/item/grenade/chem_grenade/bioterrorfoam
+	cost = 5
+	surplus = 35
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
 /datum/uplink_item/explosives/virus_grenade
 	name = "Fungal Tuberculosis Grenade"
 	desc = "A primed bio-grenade packed into a compact box. Comes with five Bio Virus Antidote Kit (BVAK) \
 			autoinjectors for rapid application on up to two targets each, a syringe, and a bottle containing \
-			the BVAK solution."
+			the BVAK solution. WARNING: EXTREME BIOHAZARD. HANDLE WITH CAUTION."
 	item = /obj/item/storage/box/syndie_kit/tuberculosisgrenade
 	cost = 8
 	surplus = 35
 	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 	restricted = TRUE
 
-/datum/uplink_item/explosives/grenadier
-	name = "Grenadier's belt"
-	desc = "A belt containing 26 lethally dangerous and destructive grenades. Comes with an extra multitool and screwdriver."
-	item = /obj/item/storage/belt/grenade/full
-	include_modes = list(/datum/game_mode/nuclear)
-	cost = 22
+/datum/uplink_item/explosives/tearstache
+	name = "Teachstache Grenade"
+	desc = "A teargas grenade that launches sticky moustaches onto the face of anyone not wearing a clown or mime mask. The moustaches will \
+		remain attached to the face of all targets for one minute, preventing the use of breath masks and other such devices."
+	item = /obj/item/grenade/chem_grenade/teargas/moustache
+	cost = 6
 	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
```

Comment:
Maybe not nukies

Reply:
Why?

Reply:
Its more of a clown ops rather then any old nukie, its really powerful when dealing with cap in space as it pends armor as well as it not protecting nukies that walk into the gas

Reply:
It seems a bit buggy and drains stam hard. I am up for removing it from nukies.